.github/workflows/continuous-delivery.yaml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Delivery
+
+on:
+  push:
+    branches:
+      - '*.x' # maintenance releases such as 15.x
+      - beta
+      - master
+      - next
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache npm dependencies
+        uses: actions/cache@v1
+        with:
+          key: npm-${{ hashFiles('package-lock.json') }}
+          path: ~/.npm
+          restore-keys: |
+            npm-
+      - name: Install dependencies
+        run: npm ci --ignore-scripts --no-audit --no-progress --prefer-offline
```

Comment:
`--ignore-scripts` for additional security, in case we install some malicious packages that could leak tokens through the use of scripts

`--no-audit --no-progress --prefer-offline` for performance